packages/flutter/test/widgets/widget_inspector_structured_error_test.dart

```diff
@@ -0,0 +1,63 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'widget_inspector_test_utils.dart';
+
+void main() {
+  StructuredErrorTestService.runTests();
+}
+
+class StructuredErrorTestService extends TestWidgetInspectorService {
+  @override
+  bool isStructuredErrorsEnabled() {
+    return true;
+  }
+
+  static void runTests() {
+    final StructuredErrorTestService service = StructuredErrorTestService();
+    WidgetInspectorService.instance = service;
+    FlutterExceptionHandler testHandler;
+    FlutterExceptionHandler setUpAllHandler;
+
+    setUpAll(() {
+      setUpAllHandler = FlutterError.onError;
```

Comment:
give a name that describes what this is not where it came from.
maybe: inspectorServiceErrorHandler.